pkgs/by-name/pr/procfd/package.nix

```
@@ -0,0 +1,28 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "procfd";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "deshaw";
+    repo = "procfd";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-M2VFy7WqvoxgzEpS0qd7nGLRt2pKGZlfU9uUHlwDC7Y=";
+  };
+
+  cargoHash = "sha256-YmUzcJ8SM3iwjeDZXWBrDcT793ZyF6QdwxuYDh69xZw=";
+
```

```suggestion

  nativeInstallCheckInputs = [
    versionCheckHook
  ];
  versionCheckProgramArg = "--version";
  doInstallCheck = true;

```
Can you try adding this please?

---

Is it fine to skip setting `versionCheckProgramArg` as it defaults to `--help` and then `--version` anyway?